src/main/haskell/language-kore/test/parser/Data/Kore/ASTVerifier/DefinitionVerifierSortUsageTest.hs

```diff
@@ -184,10 +184,11 @@ definitionVerifierSortUsageTests =
                 { testConfigurationDescription = "Correct sort count"
                 , testConfigurationAdditionalSentences =
                     [ simpleSortSentence additionalSortName
-                    , ObjectSentence $ SentenceSortSentence SentenceSort
-                        { sentenceSortName = Id "UnarySort"
+                    , asSentence SentenceSort
+                        { sentenceSortName = Id "UnarySort" :: Id Object
```

Comment:
I'd annotate the SentenceSort with ":: SentenceSort Object" because it says what you mean, instead of implying what you mean as the Id annotation does.

This pattern repeats a lot, so I'll let you choose if you change it or not.

Reply:
I think I thought about that, originally.  One issue though with that approach is that `SentenceSort` has 3 parameters and they would all need to be instantiated `SentenceSort level pat variable`.  Admittedly, we could probably have type aliases of the form `CommonKoreSentenceSort level` and `CommonPureSentenceSort level` which would default the other two arguments to their corresponding values.  If you think we should do this, i can try implementing such a thing.